feeder/cmd/pixel_bt_feeder/README.md

```diff
@@ -0,0 +1,18 @@
+# Pixel BT feeder
+
+This feeder knows how to interact with the
+[Pixel BT log](https://developers.devsite.corp.google.com/android/binary_transparency/pixel)
+to fetch checkpoints and proofs to feed to the [generic witness](/witness/golang).
+
+> :frog:
+> Note that while the Pixel BT checkpoint _body_ is compatible with the common checkpoint
+> format described in [/formats/log](/formats/log), currently, the signature is not. So
```

Comment:
Worth being clear that it's the signature encoding format that is problematic, but the signature algorithm is OK?

Reply:
Great call, thanks - done!